python/lsst/ip/diffim/getTemplate.py

```diff
@@ -719,18 +755,18 @@ def getDcrModel(self, patchList, coaddRefs, visitInfo):
         Parameters
         ----------
         patchList : `dict`
-            Dict of the patches containing valid data for each tract
-        coaddRefs : `list` of elements of type
-                    `lsst.daf.butler.DeferredDatasetHandle` of
-                    `lsst.afw.image.Exposure`
+            Dict of the patches containing valid data for each tract.
+        coaddRefs : `list`
```

Comment:
Have you run `package-docs build` on this branch? This multi-line listing of types makes sphinx very upset. I think you need to instead do something like

```python
coaddRefs : `list` [`lsst.daf.butler.DeferredDatasetHandle`]
    Data references to `~lsst.afw.Image.Exposure` representing DcrModels
    that overlap the detector.
````

Reply:
So, no, I had not...but I have now.  I admit to knowing nothing about sphinx, so am happy to go with your suggestion.  For the record, I ran `package-docs build` on the branch as is and copied the `doc` folder to my local machine and looked at it via opening `doc/_build/html/index.html` and here's what it looks like:
![docsDM-30927_getDcrModel](https://user-images.githubusercontent.com/10424871/190948269-0167846b-fe8a-4cfb-85bb-19477fc493d7.png)
and with your edit:
![docsTimEdit_getDcrModel](https://user-images.githubusercontent.com/10424871/190948369-2624be93-2609-4dd5-929e-5f5844ab1bcd.png)
Either works for me!